include/seqan/platform/platform_gcc.h

```diff
@@ -39,9 +39,20 @@
  * @signature #define PLATFORM_GCC
  */
 
+// GNU COMPATIBLE
 #ifndef PLATFORM_GCC
 #define PLATFORM_GCC
 
+#if !defined(__clang__) && !defined(__ICC) && !defined(PLATFORM_GNU)
+// ACTUALLY GNU
+#define PLATFORM_GNU
+#endif
+
+#if defined(__clang__) && !defined(__ICC) && !defined(PLATFORM_CLANG)
```

Comment:
True, but I thought it would make more sense that we use our own DEFs throughtout. We also define PLATFORM_WINDOWS_VS, although detecting MSC_VER would suffice.